### PR TITLE
Use NautyTracesInterface

### PR DESCRIPTION
**Pull request by james-d-mitchell**
This PR allows the use of Nauty-Traces via the NautyTracesInterface package in Digraphs. 

The documentation is missing, or out of date, but otherwise I think this is complete.

### CODE REVIEW

**Comment by james-d-mitchell** [comment]
Looks like NautyTracesInterface doesn't work in 32 bit mode

**Comment by wilfwilson** [comment]
How shall we resolve the problem with 32-bit mode? Certainly we don't want Travis to try and run tests involving NautyTracesInterface when in 32-bit mode. Is that all we need to do?

**Comment by sebasguts** [comment]
I am currently looking at the 32-bit mode problems, and hopefully solve them in NautyTracesInterface.

**Comment by james-d-mitchell** [comment]
This requires a bit more work before it can be merged, but seems to work ok now, thanks @sebasguts

**Comment by james-d-mitchell** [comment]
This PR resolves Issue #97

File: gap/isomorph.gi [hunk 1]
```diff
@@ -0,0 +1,556 @@
...
+##
+
+# This file contains methods using bliss or nauty, for computing isomorphisms,
+# automorphisms, and canonical labellings of digraphs.
+
+InstallGlobalFunction(DigraphsUseBliss,
+function()
+  # Just do nothing if NautyTracesInterface is not available.
+  if DIGRAPHS_NautyAvailable then
+    Info(InfoWarning,
```
Comment (wilfwilson) [inline gap/isomorph.gi]: Why are you using `InfoWarning` here (and elsewhere) rather than `InfoDigraphs`?

File: doc/isomorph.xml [hunk 2]
```diff
@@ -1,14 +1,95 @@
...
+gap> BlissAutomorphismGroup(JohnsonDigraph(5, 2));
+Group([ (3,4)(6,7)(8,9), (2,3)(5,6)(9,10), (2,5)(3,6)(4,7), (1,2)(6,8)
+(7,9) ])]]></Example>
+    </Description>
+  </ManSection>
+<#/GAPDoc>
+
+<#GAPDoc Label="NautyAutomorphismGroup">
+<ManSection>
+  <Attr Name="NautyAutomorphismGroup" Arg="digraph[, colours]"/>
```
Comment (wilfwilson) [inline doc/isomorph.xml]: Is it true that nauty can only handle digraphs without multiple edges? If so, this should be documented in all the relevant places.

**Comment by james-d-mitchell** [comment]
This was intentional, as I'd like to make a PR soon to withdraw multidigraphs, and as such I didn't think it was worth writing documentation for a feature, only to remove it almost immediately. What do you think @wilfwilson ?

**Comment by wilfwilson** [comment]
That's fine.

**Comment by james-d-mitchell** [comment]
So, merge or wait?

**Pull request merged**